crates/testsuite/features/cmd.feature

```diff
@@ -54,7 +54,8 @@ Feature: Rooch CLI integration tests
 
       # session key
       Then cmd: "session-key create  --app-name test --app-url https:://test.rooch.network --scope 0x3::empty::empty"
-      Then cmd: "move run --function 0x3::empty::empty  --session-key {{$.session-key[-1].authentication_key}}"
+      Then cmd: "session-key list"
```

Comment:
Add an assert to ensure the command is executed successfully.

Reply:
The error message is like this.
```
rooch session-key list
Error: Session key not found
```
assert this
Then assert: "'{{$.session-key[-1]}}' not_contains Error"